tests/physics/controllers/rolling_friction_supervisor/rolling_friction_supervisor.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 1996-2021 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <webots/robot.h>
+#include <webots/supervisor.h>
+
+#include "../../../lib/ts_assertion.h"
+#include "../../../lib/ts_utils.h"
+
+#define TIME_STEP 32
+
+int main(int argc, char **argv) {
+  ts_setup(argv[0]);
+
+  char name[8];
+  const double lateral_velocity[6] = {0, 0, 0, -25, 0, 0};
+  const double spinning_velocity[6] = {0, 0, 0, 0, 0, 25};
+
+  WbNodeRef ball_nodes[9];
```

Comment:
Instead of hard-coding 9 everywhere, we could define it as a constant along the TIME_STEP value:

```C
#define BALL_COUNT 9
```